extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss

```diff
@@ -101,7 +101,7 @@ $start-a-wikia-logo-size: 30px;
 		}
 	}
 
-	@media #{$breakpoint-small-only} {
+	@media #{$breakpoint-below-small-plus} {
```

Comment:
why not moving it to breakpoints-layout.scss so all breakpoint related code is the same place. Next time similar issue happens we know where to look for.